Project-Uni/RootShare
# Navigate User to Mux Stream on Event Start

**Updates**
1. Navigate user to mux stream on event start
2. Connect Webinar Cache to DB

**Testing**
- Navigate to a valid event page as a viewer
- In a separate window, navigate to event page as a host
- Press begin live streaming
- In 45 seconds, viewer window should receive an alert that stream has started and be shown the mux playback stream 
- When you click manage speakers as the host, that user should appear in the list
